.github/pull_request_template.md

```diff
@@ -4,6 +4,6 @@
 
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
-- [ ] I have checked that [readthedocs](https://readthedocs.org/projects/credoai-lens/) API still is constructed correctly
+- [ ] I have checked that [readthedocs](https://readthedocs.org/projects/credoai-lens/) API documentation is constructed correctly
 - [ ] I have built tests for new functionality (particularly new evaluators)
```

Comment:
I think we should try to be more comprehensive about what tests we write. We can push for that a bit in this review checklist.

Right now I think our internal default expectation is to add a `add()` and a `run()` call to the test suite. For evaluators we should also do `to_evidence()`, `send_to_governance()`, and `export()`; all of those can be put in a testing checklist for the PR.
- It might also be good to have the person who opens the PR to write up a summary of edge cases they thought of and wrote test cases for. This will be good for forcing both the PR-opener and the reviewer to think more expansively about ways things could go wrong. I think the default right now is to primarily try to assess whether the code will solve exactly the narrow issue it's designed to.

Reply:
This is more expansive and test-heavy than I think we need to be right now.